### ISSUE TITLE
Error compiling makefile_windows

### ISSUE THREAD

**Issue by ntahlah** [vedaldi/svm-struct-matlab#8]
Hello! 

I've been encountered few errors when I executing makefile_windows which as you can see below: 

doing hideo 
Building with 'Microsoft Visual C++ 2013 Professional (C)'.
Error using mex
svm_hideo.c
c:\users\ibrahimpc2\documents\matlab\svm-struct-matlab-master\svm_light\svm_common.h(447) : error C2059: syntax error : 'sizeof'

Error in makefile_windows (line 3)
mex -largeArrayDims  -c  -DWIN ./svm_light/svm_hideo.c

Can somebody guide me on how to solve it?

**Comment by brisker** [comment]
Have you solve this problem?

**Comment by ntahlah** [comment]
Yes, I have. Apparently one of my mate play around with it and voila suddenly it can run.

**Comment by jialin666** [comment]
Hi, ntahlah， how did you solved the problem? Could you please tell me? I have the same problem with you.

**Comment by shaofengzeng** [comment]
Hi, ntahlah, i also have this problem, can you tell me how to solve it ?

**Comment by shaofengzeng** [comment]
If you have this problem, just comment the line  447 in svm_light/svm_common.h  //int isnan(double a);
and lines 1773-1776 in svm_light/svm_common.c

**Comment by jizhihang** [comment]
Just comment the line about "isnan"  in svm_light/svm_common.h  and svm_light/svm_common.c

**Comment by jizhihang** [comment]
just comment the line about function "isnan" in svm_light/svm_common.h and  in svm_light /svm_common.c
